charts/hmc-hmi-outbound-adapter/values.preview.template.yaml

```
@@ -2,8 +2,23 @@ java:
   # Don't modify below here
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  environment:
+    HMCServiceBusQueueName: hmc-to-hmi-{{ .Release.Name }}
```

```suggestion
    HMCServiceBusQueueName: hmc-to-hmi
```
PR deployment will fail unless it matches